flybywiresim/a32nx
# fix: cockpit door video priority

Fixes #7168

## Summary of Changes

This fixes the priority of the cockpit door video screen.
If the video is shown and another page is selected the video will be toggled off.
## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): mozi#1454

## Testing instructions

1. Startup A32NX
2. Toggle on the VIDEO button in the pedestal panel
3. (Video should be visible on the lower ECAM)
4. Change ECAM Page
5. The video should be closed and the new page should be visible
6. Press again VIDEO button
7. The video should be visible again


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
